src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/SubmissionType.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.companieshouse.overseasentitiesapi.service;
+
+public enum SubmissionType {
+    REGISTRATION,
+    UPDATE,
+    REMOVE
```

Comment:
Minor - but since we've no remove functionality (as far as I am aware), I'd vote for not including remove here yet.